.kitchen.yml

```diff
@@ -17,16 +17,27 @@ platforms:
 suites:
 - name: delete_from_list
   run_list:
+    - recipe[test::resource_status]
```

Comment:
Removed the recipe from all of the resources except replace_or_add where it is actually used.